mne/viz/_3d.py

```diff
@@ -1718,6 +1719,323 @@ def _get_ps_kwargs(initial_time, require='0.6'):
     return initial_time, ad_kwargs, sd_kwargs
 
 
+def _glass_brain_crosshairs(params, x, y, z):
+    from matplotlib import patheffects
+    shadow = patheffects.withStroke(foreground='k', linewidth=2)
+    marker_kwargs = dict(
+        markersize=20, markerfacecolor='none', markeredgecolor='w',
+        markeredgewidth=0.5, marker='+', path_effects=[shadow], lw=1)
+    for ax, a, b in ((params['ax_y'], x, z),
+                     (params['ax_x'], y, z),
+                     (params['ax_z'], x, y)):
+        ax.plot(a, b, 'o', **marker_kwargs)[0]
+
+
+@verbose
+def plot_volume_source_estimates(stc, src, subject=None, subjects_dir=None,
+                                 mode='stat_map', bg_img=None, colorbar=True,
+                                 colormap='auto', clim='auto',
+                                 transparent=None, show=True, verbose=None):
+    """Plot Nutmeg style volumetric source estimates using nilearn.
+
+    Parameters
+    ----------
+    stc : VectorSourceEstimate
+        The vector source estimate to plot.
+    src : instance of SourceSpaces
+        The source space.
+    subject : str | None
+        The subject name corresponding to FreeSurfer environment
+        variable SUBJECT. If None stc.subject will be used. If that
+        is None, the environment will be used.
+    subjects_dir : str
+        The path to the freesurfer subjects reconstructions.
+        It corresponds to Freesurfer environment variable SUBJECTS_DIR.
+    mode : str
+        The plotting mode to use. Either 'stat_map' (default) or 'glass_brain'.
+        For "glass_brain", activation absolute values are displayed
+        after being transformed to a standard MNI brain.
```

Comment:
The time series still shows me signed values but the stat map shows absolute values. They should be made consistent, no?

Reply:
I don't think they need to be. The glass brain does abs for the MIP so we can't and shouldn't change that, and I see no reason to distort the time series unnecessarily. These seem like doc issues to me.